src/sklearn_evaluation/tracker.py

```diff
@@ -379,7 +379,30 @@ def upsert(self, uuid, parameters):
         self.conn.commit()
 
     def upsert_append(self, uuid, parameters):
-        """Append the parameters to an existing experiment"""
+        """Append the parameters to an existing experiment
+
+        Examples
+        --------
+        >>> from sklearn_evaluation import SQLiteTracker
+        >>> tracker = SQLiteTracker('experiments.db')
+        >>> exp = tracker.new_experiment()
+        >>> exp.log("accuracy", 0.8) # doctest: +SKIP
+        0.8
+        >>> tracker.upsert_append(exp.uuid, dict(accuracy=0.9, loss=[0.4, 0.2]))
+        >>> data = tracker.get(exp.uuid)._data
```

Comment:
```
>>> data = tracker.get(exp.uuid)._data
        >>> data # doctest: +SKIP
        {'accuracy': [0.8, 0.9], 'loss': [0.4, 0.2]}
```

This can be removed since the latter `tracker.query` is displaying the same thing.

Also, please use some other variables than `loss` and `accuracy`. It might be confusing since both these should be lists ideally.